concert/base.py

```diff
@@ -1099,6 +1099,18 @@ def set_parameter(instance, parameter_value, wait_on=None):
         if not hasattr(self, '_get_target_' + param.name):
             setattr(self, '_get_target_' + param.name, _getter_target_not_implemented)
 
+    def _remove_parameter(self, param):
```

Comment:
What about lingering `obj.get_parameter` and the same for set. Could that be an issue. The removal should bring the class to the same state as before the parameter installation. This should also be tested.

Reply:
How about now?

Reply:
I still find it weird that you need to remove properties. It is an indication that the base class `Radiography` is not general enough. On the other hand, couldn't you just use the number of flats and darks as `num_steps_per_period`, isn't that kind of the number of projections? And you could set it separately for flats and radios...

Reply:
How about leaving the `_remove_parameter` in if we need in in the future?